api/src/Controller/BaseController.php

```diff
@@ -38,6 +38,53 @@ class ConflictException extends Exception
 
 }
 
+class IncludeResource
```

Comment:
Not a stopper for this PR, but ideally we should be one class per file. It's not really a big deal, though.